Project.toml

```diff
@@ -1,10 +1,11 @@
 name = "ONSAS"
 uuid = "7d8da659-23aa-453a-9a19-a3d1f6c795f5"
 authors = ["M.C. Vanzulli <mvanzulli@fing.edu.uy>", "M. Forets <mforets@gmail.com>", "J. M. Pérez Zerpa <jorgepz@fing.edu.uy>"]
-version = "0.3"
+version = "0.3.0"
```

Comment:
Nice, after closing LinearSolve.jl we can make a release and then continue with the cache #413